packages/client/utils/GitHubManager.ts

```diff
@@ -22,7 +22,7 @@ type DocVariables = any
 // type DocVariables<T> = T extends DocumentNode<any, infer V> ? V : never
 
 abstract class GitHubManager {
-  static SCOPE = 'admin:org_hook,read:org,repo,user:email,write:repo_hook'
+  static SCOPE = 'admin:org_hook,read:org,repo,user,write:repo_hook'
```

Comment:
Updated the scope so that users can see contributions to public & private repos. The old scopes would only show contributions to public repos. I think this means that we need to remove users' GitHub auth so that they accept the new scope?

Reply:
oof, yessir.
we've been talking about it for years, but i think with `GitHubAuth` living in postgres, we should probably use it as an example:
the `GitHubAuth` object should contain a `scope`. that way, we can compare the scope we have to the scope that the app needs & determine when the scope is stale. I'll make an issue & see if we can sneak it in in a timely manner